src/SonarScanner.MSBuild.PreProcessor/JreCaching/JreCache.cs

```diff
@@ -145,7 +145,19 @@ private async Task<JreCacheResult> UnpackJre(string downloadTarget, JreDescripto
         {
```

Comment:
The method is async without awaiting and requires the ``cacheRootLocation` that is not used.

Is this because we expect to need them for the unpacking implementation?

Reply:
Yes, I think so.
I had the same question, but I did not want to change the signature of the method, since Martin is currently working on it.

Reply:
I thought that the file I/O will be async, but the libraries don't seem to support it. We may remove the async part if it isn't used by any of the implementations. (e.g. for the zip, there is not async version: https://learn.microsoft.com/en-us/dotnet/api/system.io.compression.zipfile.extracttodirectory?view=net-8.0)